proprietary/tokens/src/components/ams/dialog.tokens.json

```diff
@@ -3,12 +3,13 @@
     "dialog": {
       "background-color": { "value": "{ams.color.primary-white}" },
       "border": { "value": "0" },
-      "max-inline-size": { "value": "min(87.69vw, 45rem)" },
+      "inline-size": { "value": "calc(100vw - 2 * {ams.space.grid.md})" },
+      "max-block-size": { "value": "calc(100vh - 2 * {ams.space.grid.md})" },
```

Comment:
Shouldn't Dialog just use `%` instead of `vw` / `vh`? It's ok for it to take up a certain portion of space relative to its parent, no? Currently it listens to the viewport, which means it overflows when used in a smaller container (like currently in the Storybook docs on smaller screens). Added benefit: we would be rid of 2 more directional values.

Reply:
A percentage width requires a Dialog to be rendered at a position in the DOM where no parents can constrain the width. I don’t think that’s a sacrifice we can make. With viewport units, people can render the Dialog anywhere they want.

Reply:
Or `100dvh` to prevent weird overflow problems on mobile devices?